#### Problem Statement 1:
Blood glucose levels for obese patients have a mean of 100 with a standard deviation of 15. A researcher thinks that a diet high in raw cornstarch will have a positive effect on
blood glucose levels. A sample of 36 patients who have tried the raw cornstarch diet
have a mean glucose level of 108. Test the hypothesis that the raw cornstarch had an
effect or not.

In [1]:
import pandas as pd
import numpy as np 
import math
import scipy.stats as sts
from scipy.stats import norm

In [2]:
pop_mean = 100
pop_std = 15
n = 36
sample_mean = 108
alpha = 0.05

SE = pop_std/n**0.5

Z = (sample_mean - pop_mean)/SE
print("Z score is: {}".format(Z))


Z score is: 3.2


From Z table probability of having value less than 108 is 0.9993 and more or equal to 108 probability equal to 0.0007
Because probability of having mean glucoselevel more than or equal to 108 is less than 0.05 we can reject the null hypothesis there is a raw cornstarch effect. 

### Problem Statement 2:
In one state, 52% of the voters are Republicans, and 48% are Democrats. In a second
state, 47% of the voters are Republicans, and 53% are Democrats. Suppose a simple
random sample of 100 voters are surveyed from each state.
What is the probability that the survey will show a greater percentage of Republican
voters in the second state than in the first state?

In [4]:
n1 = 100
n2 = 100
rep1 = 0.52
dem1 = 0.48
rep2 = 0.47
dem2 = 0.53

mu = rep1 - rep2
print('Mu is : {}'.format(mu))

std = math.sqrt(((rep1*dem1)/n1) + ((rep2*dem2)/n2))
x = 0
Z = (x - mu)/std
print("Z score is: {}".format(Z))

Mu is : 0.050000000000000044
Z score is: -0.7080278164104213


From Z score table probability for Z score -0.708 or less is 0.24916.
The probability that the survey will show a greater percentage of Republican voters in the second state than in the first state is 0.24

#### Problem Statement 3:
You take the SAT and score 1100. The mean score for the SAT is 1026 and the standard
deviation is 209. How well did you score on the test compared to the average test taker?

In [7]:
s1 = 1100
mean = 1026
std = 209

z = (s1 - mean)/std
print("Z score is: {}".format(z))
print("My Score is in the range {} - {}  with a  zscore {:.2f}".format(mean - std,mean + std,z))

Z score is: 0.35406698564593303
My Score is in the range 817 - 1235  with a  zscore 0.35
